package.json

```diff
@@ -10,22 +10,27 @@
     "atom": ">0.50.0"
   },
   "scripts": {
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "watch": "ucompiler watch src",
+    "prepublish": "ucompiler go"
```

Comment:
Also, another script named `compile` would be nice, so if someone wants to compile it real quick they can do `npm run compile`